index.bs

```diff
@@ -176,19 +185,101 @@ When an item is of type <dfn value export for="AV1 Image Item Type">av01</dfn>,
+
+[[!AV1]] delegates the selection of which [=Operating Point=] to process to the application. AVIF defines the [=OperatingPointSelectorProperty=] to control this selection. In the absence of an [=OperatingPointSelectorProperty=] associated with an [=AV1 Image Item=], the AVIF renderer is free to process any [=Operating Point=] present in the [=AV1 Image Item Data=]. In particular, when the [=AV1 Image Item=] is composed of a unique [=Operating Point=], the [=OperatingPointSelectorProperty=] should not be present. If an [=OperatingPointSelectorProperty=] is associated with an [=AV1 Image Item=], the <code>op_index</code> field indicates which [=Operating Point=] is expected to be presented for this item.
+
+NOTE: When an author wants to offer the ability to render multiple [=Operating Points=] from the same AV1 image (e.g. in the case of multi-view images), multiple [=AV1 Image Items=] can be created that share the same [=AV1 Image Item Data=] but have different [=OperatingPointSelectorProperty=]s.
+
+[[!AV1]] expects the renderer to display only one frame within the selected [=Operating Point=], which should be the highest spatial layer that is both within the [=Operating Point=] and present within the temporal unit, but [[!AV1]] leaves the option for other applications to set their own policy about which frames are output. AVIF sets a different policy, and defines how the 'lsel' property (defined in [[!HEIF]]) is used to control which layer is rendered. In the absence of a 'lsel' property associated with an [=AV1 Image Item=], the renderer is free to render either: only the output image of the highest spatial layer, or to render all output images of all the intermediate layers, resulting in a form of progressive decoding. If a 'lsel' property is associated with an [=AV1 Image Item=], the renderer is expected to render only the output image for that layer.
+
+NOTE: When such a progressive decoding of the layers within an [=Operating Point=] is not desired or when a controlled rendering of the different output images is desired (e.g. grid of layered images), multiple [=AV1 Image Items=] sharing the same [=AV1 Image Item Data=] can be created and associated with different 'lsel' properties, each with a different value of 'layer_id'. In the case of an image grid composed of 4 layered images, each made of 2 layers, controlled rendering could be achieved by creating a set of items as follows: 4 items each corresponding to the low quality/resolution image as decoded from the first layer of each of the 4 layered images, i.e. with 'layer_id' set to 0; 4 items corresponding the second layer, i.e. with 'layer_id' set to 1; a grid item composing the first set of 4 images; another grid item composing the second set of 4 images.
```

Comment:
Let me try to explain so that we can come up with a better text. 

Let's say you have a grid with 4 images, each made of only one OP and each OP made of 2 layers (increased resolution or improved quality). If the author does not provide any additional information (no lsel), according to what is in the spec, the renderer is free to display each image in the grid the way it wants. This means it could display intermediate rendering of the grid where 2 images would show the low quality and 2 would show the high quality. If the author does not want that, it could: create 2 grids. The first grid would point to 4 items that have lsel = 1 and the second grid point to 4 different items that have lsel = 2. The 2 grids could be grouped as recommended by HEIF with 'altr' grouping. 

As I write this explanation, I realize this is too complicated for an example. I simplified the example (not mentioning grids).

Reply:
Thanks for the explanation! What I don't understand is whether we want to allow a layered grid image to be decoded (or rendered) progressively. This PR describes progressive decoding only when the 'lsel' property is absent. In this grid image example, the two grid images each have an associated 'lsel' property, so they are not expected to be decoded/rendered progressively. If that is intended, then it would be good to describe another example of a grid image that can be decoded/rendered progressively.

Reply:
The grid images would not have the 'lsel' property because they are not coded images, they don't have layers. Only the referenced images have layers and can have 'lsel'. 

Now that the example is removed, do you think we should have a grid example?

Reply:
I don't think we need a grid example. If we add the grid example, a good place to add it may be near this NOTE at line 212 because it is similar to what the NOTE describes:

`NOTE: When such a progressive decoding of the layers within an [=Operating Point=] is not desired or when an author wants to expose each layer as a specific item, multiple [=AV1 Image Items=] sharing the same [=AV1 Image Item Data=] can be created and associated with different 'lsel' properties, each with a different value of 'layer_id'.`